src/Ninja/G1CP/Content/Fixes/Gamesave/fix203_LogEntryGrahamMerchant.d

```diff
@@ -0,0 +1,149 @@
+/*
+ * #203 Graham not listed as merchant
+ */
+
+/*
+ * Since applying and reverting both depend on finding the necessary symbols this is a toggle function
+ */
+func int G1CP_203_LogEntryGrahamMerchant_Toggle(var int apply) {
+    // Define possibly missing symbols locally
+    const int LOG_MISSION = 0;
+    const int LOG_NOTE    = 1;
+
+    // Parameters of this fix
+    const int    topicSection  = LOG_NOTE;
+    const string topicSymbName = "GE_TraderOC";
+    const string infoSymbName  = "DIA_Graham_Hello";
+    const string funcSymbName  = "DIA_Graham_Hello_Info";
+    const string hookSymbName  = "G1CP_203_LogEntryGrahamMerchant_Intercept";
+
+    // Retrieve the topic and entry strings once and modify the info function
+    const int    infoId  = -2; // -1 is reserved for invalid symbols
+    const int    topicId = -1;
+    const int    entryId = -1;
+    const string topic   = "G1CP invalid topic name";
+    const string entry   = "G1CP invalid topic entry";
+    if (infoId == -2) {
+        infoId = MEM_GetSymbolIndex(infoSymbName);
+        topicId = MEM_GetSymbolIndex(topicSymbName);
+        var int funcId; funcId = MEM_GetSymbolIndex(funcSymbName);
+        var int b_logentry_id; b_logentry_id = MEM_GetSymbolIndex("B_LogEntry");
```

Comment:
These lines are something we can refactor soon. Let's merge this PR first, and then do the entire scripts at once. To use more specific functions instead of `MEM_GetSymbolIndex`.